pkgs/racket-test-core/tests/racket/hash.rktl

```diff
@@ -3,7 +3,7 @@
 
 (Section 'hash)
 
-(require racket/hash)
+(require racket/hash (only-in racket list? number? null? symbol? eq? eqv?))
```

Comment:
I don't think this is needed. They are all provided in the `base` environment already.